igvm/hypervisor.py

```diff
@@ -817,3 +832,57 @@ def device_to_netcat(self, device, size, listener):
             '| /bin/nc.openbsd -q 1 {2} {3}'
             .format(device, size, *listener)
         )
+
+    def hv_pcpus_used_by_vm(self, vm) -> float:
+        """pCPUs used by VM
+
+        Calculate the physical CPUs the VM will use on a potential destination
+        hypervisor.
+
+        :param: vm: VM object
+
+        :return: Amount of pCPUs as float
+        """
+
+        vm_performance_value = vm.vm_performance_value()
+
+        hv_perffactor_dest = self.dataset_obj['cpu_perffactor'] / 1000
+        hv_pcpus_dest = vm_performance_value / hv_perffactor_dest
+
+        return hv_pcpus_dest
```

Comment:
You are not explicit casting to a float. The result of the division might return a integer in case the numbers are even like e.g. 4 / 2 = 2 is this a problem ? (You may want to check the following methods as well)

Reply:
Now everything is casted as `float` explicitly.